packages/android_intent/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,14 +10,6 @@
     android:icon="@mipmap/ic_launcher"
     android:label="android_intent_example"
     android:name="io.flutter.app.FlutterApplication">
```

Comment:
nit: This line was removed for the rest of the plugins. Does it need to be removed here too?

Reply:
fixed. I also saw additional ones. thanks